web/frontend/src/components/navigation/hooks/use-nav-client.ts

```diff
@@ -5,4 +5,4 @@ import { rcLogConditionally } from '@/lib/log';
 export const useNavClient = (name: string) => {
   const { robotClient } = useRobotClient();
   return new NavigationClient(robotClient.current, name, { requestLogger: rcLogConditionally });
-};
+};
```

Comment:
[Nit] looks like a few unintentional removals of trailing newlines got into this diff, should they be reverted?

Reply:
I undid the formatting changes I am guessing this is a consequence of that, I will just revert. These kind of things are going to come up every PR unless we decide to just allow the formatting changes as we go, or commit a formatting change PR.

Reply:
I'm up for adding prettier to RC after this is merged

Reply:
Yeah hard agree that a add-formatting-only PR is the way to go. In the absence of a prettier config, allowing PRs to update unrelated formatting from each developer's editor (which may have different settings) is disruptive and hides bugs

Reply:
We could tackle this with R2D2 control tab work.